pkg/webhooks/validation/controller/controller.go

```diff
@@ -107,124 +104,70 @@ func NewValidatingWebhookController(client kube.Client,
 		CABundleWatcher:  caBundleWatcher,
 		Revision:         revision,
 		ServiceName:      "istiod",
+		Backoff:          time.Second,
+		Retries:          5,
```

Comment:
This can make it unrecoverable, we an make it max int like injector webhook

Reply:
ack